pkg/stream/message_test.go

```diff
@@ -0,0 +1,174 @@
+/*
+Copyright 2020 The KubeEdge Authors.
```

Comment:
Copyright date should be 2023 : )
Then /lgtm

Reply:
updated